.github/workflows/README.md

```diff
@@ -0,0 +1,34 @@
+# Check projects with Spotbugs Github Action
+
+[Github Actions](https://docs.github.com/en/actions)  can be used to automate CI/CD activities.
+A series of such action is called a workflow.
+This repository contains 2 workflows which can be used to automatically run a Spotbugs Java projects.
```

Comment:
"projects" -> "project"

Reply:
Actually we already has workflows in `.github/workflows` so "This repository contains 2 workflows" sounds strange. Is it OK to move these explanation into YAML file itself?